### ISSUE TITLE
[4.1] Accessibility plugin is not accessible

### ISSUE THREAD

**Issue by drmenzelit** [joomla/joomla-cms#37210]
### Steps to reproduce the issue
Activate the plugin System - Additional Accessibility Features, no matter if backend, frontend or both.
Try to access the button using the hotkey CTRL + ALT + A (Windows) or using the keyboard.

### Expected result
You can open and operate the function from the plugin without a mouse.

### Actual result
It is impossible to reach the button using the keayboard. The hotkey is not working either (tested in Firefox and Chrome on Windows).

### System information (as much as possible)
Joomla 4.1.1-dev

### Additional comments
The icons look very weird, that seems to be related with this PR: https://github.com/joomla/joomla-cms/pull/36948
![grafik](https://user-images.githubusercontent.com/9153168/156927939-3f1f3794-e081-4243-b13c-514ae8f48b9a.png)

The wheel chair icon should be replaced by the newer accessibility symbol 
![grafik](https://user-images.githubusercontent.com/9153168/156927898-c55c42c7-c63d-4a63-8b7d-dc14a7184ce0.png)

**Comment by brianteeman** [comment]
Do these work for you? (They do for me)

<html>
<body>


Module or Function | Hotkey
-- | --
Toggle Menu | CTRL + ALT + A
Gray Hues | CTRL + ALT + G
Underline Links | CTRL + ALT + U
Big Cursor | CTRL + ALT + C
Reading Guide | CTRL + ALT + R
Text To Speech | CTRL + ALT + T
Speech To Text | CTRL + ALT + S


</body>
</html>

**Comment by drmenzelit** [comment]
CTRL + ALT + A -> doesn't work
CTRL + ALT + G -> is deactivated in Firefox, it works in Chrome
CTRL + ALT + U -> it works
CTRL + ALT + C -> it works
CTRL + ALT + R -> it works
CTRL + ALT + T -> it works
CTRL + ALT + S -> is deactivated

One can see the hotkeys in the code, but they are not explained in the plugin.

**Comment by brianteeman** [comment]
they are explained in the plugin upstream documentation. I cant explain why ctr-alt-a doesnt work for you as I cant replicate it. Everything else that is wrong with this plugin is because the JAT at the time insisted on having this type of plugin and after initial implementation changed their minds and then subsequently people forced incorrect changes upstream which broke everything.

**Comment by chmst** [comment]
CTRL + ALT + A opens the skip-to-nativation here but not the accessibility plugin.  (win 11)
I have all a11y plugins enabled  in backend and frontend. Maybe this causes conflicts.

**Comment by cyrez** [comment]
> The wheel chair icon should be replaced by the newer accessibility symbol

👍 

I could create a PR to update the icon, but i don't find anymore the media folder in github repository...?

**Comment by brianteeman** [comment]
> CTRL + ALT + A opens the skip-to-nativation here but not the accessibility plugin. (win 11)

That is weird - it should be alt+9

**Comment by Alam9835** [comment]
I want to work on this issue.

**Comment by drmenzelit** [comment]
Iit is not necessary to ask for permission to work on an issue.
@Alam9835 if you are a GSoC student you can join the accessibility project.

**Comment by chmst** [comment]
Additional info: 
The key combination of Control + Option + A is already reserved on a Mac by the VoiceOver application and is set to make VoiceOver start reading from where the cursor / pointer is located.

**Comment by h-tagliato** [comment]
Really exhaustive.
Thank you very much for your explanations<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37210">issues.joomla.org/tracker/joomla-cms/37210</a>.</sub>

**Comment by ahotzler** [comment]
Slightly OT: In the screenshot above, the icons are inappropriate in my eyes. 

Is that supposed to be so that a steering wheel can be seen with "Invert Colours"? The "100" at "Underline left" also makes 0 sense.

I notice this on some pages, moreover the representation of the shown icons is different on different computers (sometimes colored, sometimes gray).

The screenshot shows the output when the plugin is set to the option "Emojis", here no Google Material Fonts are loaded.

If you also see this as inappropriate, I would report a bug.

Translated with www.DeepL.com/Translator (free version)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37210">issues.joomla.org/tracker/joomla-cms/37210</a>.</sub>

**Comment by drmenzelit** [comment]
We are aware of the emojis. Since the plugin loads an external library, we can't directly change it. This year we had a GSoC student working on the plugin. We hope to bring some improvements in the next Joomla version. And maybe we would be able to use an own set of SVG icons instead of the emojis.

**Comment by ahotzler** [comment]
Ok thanks, if I can add anything with my humble knowledge let me know.

**Comment by sandewt** [comment]
Related #39615

**Comment by Quy** [comment]
This will be fixed with #43143.
